converter/src/test/resources/negative/angerClinicalDocumentValidations.xml

```diff
@@ -224,6 +193,18 @@
 							</item>
 						</list>
 					</text>
+					<entry typeCode="DRIV">
+						<act classCode="ACT" moodCode="EVN">
+							<templateId root="2.16.840.1.113883.10.20.17.3.8"/>
```

Comment:
Why did we move the reporting parameters act towards the bottom instead of the top?

Reply:
Reporting parameters was in its own section and not under a specific measure (ACI, IA, ecqm) section. The section itself wasn't being read to show that a reporting parameter start/end dates were missing so I moved it into the correct location to receive the right errors.